naiserator.yaml

```diff
@@ -31,6 +31,8 @@ spec:
         mountPath: /secrets/serviceuser/srvsblarbeid
       - kvPath: kv/##VAULT_KV_ENV##/fss/meldekortservice/##NAMESPACE##
         mountPath: /var/run/secrets/nais.io/vault
+      - kvPath: /azuread/data/##VAULT_KV_ENV##/creds/meldekortservice
+        mountPath: /var/run/secrets/nais.io/azuread
   replicas:
```

Comment:
Egenskapene for replicas/resources bør vi se på etter hvert. Tipper dette kan nedjusteres ganske mye.